lib/ukdebug/include/uk/gdb.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2021, Karlsruhe Institute of Technology. All rights reserved.
+ * Copyright (c) 2024, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UKDEBUG_GDB_H__
+#define __UKDEBUG_GDB_H__
+
+/**
+ * Set up the gdb stub and break so that the gdb host can connect
+ *
+ * This function is intended to be the first time the gdb stub is set up and
+ * entered. If the setup is successful, this function will execute a breakpoint
+ * instruction to give the gdb host a chance to connect to the stub.
+ */
+void uk_debug_gdb_early_init(void);
```

Comment:
As discussed offline, let us remove this for now since early init debugging can be a bit complicated due to order dependencies during early system initialization. This header is also not be needed anymore.